test/test-applications/integrations/TestApplication.Wcf.Server.NetFramework/StatusService.cs

```diff
@@ -0,0 +1,44 @@
+// <copyright file="StatusService.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.ServiceModel;
+using System.Threading.Tasks;
+using TestApplication.Wcf.Shared;
+
+namespace TestApplication.Wcf.Server.NetFramework;
+
+[ServiceBehavior(
+    Namespace = "http://opentelemetry.io/",
+    ConcurrencyMode = ConcurrencyMode.Multiple,
+    InstanceContextMode = InstanceContextMode.Single,
+    UseSynchronizationContext = false,
+    Name = "StatusService")]
+public class StatusService : IStatusServiceContract
+{
+    public static int TimesHit { get; set; }
+
+    public Task<StatusResponse> PingAsync(StatusRequest request)
+    {
+        TimesHit += 1;
```

Comment:
It won't be a problem with current test usage since clients are not concurrent but it gives peace of mind putting a lock around TimesHit (or the other alternatives to ensure it is correct on concurrent usage).

Reply:
Well this is just test app for tests, so I don't know if that is necessary, but I can do that :)